tests/testJointcal_lsstSim.py

```diff
@@ -0,0 +1,134 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import unittest
+import os
+
+from lsst.afw import geom, coord
+import lsst.utils
+import lsst.pex.exceptions
+from lsst.daf import persistence
+from lsst.jointcal import jointcal
+import jointcalTestBase
+
+try:
+    data_dir = lsst.utils.getPackageDir('validation_data_jointcal')
+    os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(data_dir, 'twinkles1_and_index')
+except lsst.pex.exceptions.NotFoundError:
+    data_dir = None
+
+# We don't want the absolute astrometry to become significantly worse
+# than the single-epoch astrometry (about 0.040").
+absolute_error = 42e-3/jointcalTestBase.arcsec_per_radian
```

Comment:
Totally agreed about the rationale, just wasn't sure what to say. I've added some comments about it.